### PR TITLE
Close out final 2020 Fonts Chapter TODOs

### PR DESCRIPTION
**Pull request by bazzadp**
Follow on from #1816 

Makes progress on #1432 and #902

### CODE REVIEW

File: src/content/en/2020/fonts.md [hunk 1]
```diff
@@ -20,7 +20,6 @@ featured_stat_2: 10.3%
 featured_stat_label_2: Usage of <code>font-display</code> swap
 featured_stat_3: 11.0%
 featured_stat_label_3: Usage of Variable Fonts on mobile sites
-unedited: true
```
Comment (bazzadp) [inline src/content/en/2020/fonts.md:23]: Put this back if we don't finish out the editing of this chapter in this PR (but let's try to do that!)

File: src/content/en/2020/fonts.md [hunk 2]
```diff
@@ -66,7 +65,7 @@ The country that uses the most font bytes is South Korea, which is not all that
...
 
-{# TODO (authors) - this sentence is a bit hanging our there. What's great about this thread compared to what we are covering here? Is it that you're wanting to credit that thread as the basis for the queries used by this chapter (if so say so), or you believe they have more insights than you have here (again in which case say so, but reading through the thread doesn't seem that much more, and even if there was could it not be summarised more here?), or is it you're wanting to direct people to there for further discussions? #}
+{# TODO (authors) - this sentence is a bit hanging out there. What's great about this thread compared to what we are covering here? Is it that you're wanting to credit that thread as the basis for the queries used by this chapter (if so say so), or you believe they have more insights than you have here (again in which case say so, but reading through the thread doesn't seem that much more, and even if there was could it not be summarised more here?), or is it you're wanting to direct people to there for further discussions? #}
```
Comment (bazzadp) [inline src/content/en/2020/fonts.md:68]: @raphlinus / @jpamental thoughts?

File: src/content/en/2020/fonts.md [hunk 3]
```diff
@@ -107,14 +106,12 @@ It was interesting to note the differences in speed for sites using the various
 
 When viewing commercial services such as Adobe (`use.typekit.net`) or Monotype (`fast.fonts.com`) it is interesting to note that on desktop they tend to be as fast or slightly faster than Google Fonts, but are noticeably slower on mobile. Conventional wisdom has generally held that the tracking scripts used by those services substantially slow them down, but that is apparently less an issue today than it has been in years past.
 
-{# TODO authors - Do we want to add a note to say that we are measuring site performance and not necessarily font hoster performance? It could be that Adobe sites are all just pooor on mobile and it's nothing to do with their hosting platform #}
+{# TODO authors - Do we want to add a note to say that we are measuring site performance and not necessarily font hoster performance? It could be that Adobe sites are all just pooor on mobile and it's nothing to do with their hosting platform. #}
```
Comment (bazzadp) [inline src/content/en/2020/fonts.md:109]: @raphlinus / @jpamental thoughts?

Happy enough to delete this TODO if you don't think anything further is needed here.

File: src/content/en/2020/fonts.md [hunk 4]
```diff
@@ -107,14 +106,12 @@ It was interesting to note the differences in speed for sites using the various
...
 
 #### Self-hosting isn't always better
 
 Self-hosting fonts on the same domain as the website can be faster, as [we discovered for this very website](https://www.tunetheweb.com/blog/should-you-self-host-google-fonts/), however this is not always the case as the data shows.
 
-{# TODO Authors - Suggested different wording here, and moved `local` down to below. Let me knwo your thoughts. Particularly as I inserted a reference to my own site here which is a bit presumptious of me :-) but it is about the Web Almanac so think it fits. #}
```
Comment (bazzadp) [inline src/content/en/2020/fonts.md:116]: Not sure if you both got a chance to review my edits here but removed this TODO under the assumption you're OK with them. Let me know if you want any further changes here. Or just resolve this comment if happy.

File: src/content/en/2020/fonts.md [hunk 5]
```diff
@@ -329,9 +324,9 @@ Lots more information about color fonts, including examples, are available at [c
 
 Variable fonts are certainly one of the biggest stories this year. They're seen in 10.54% of desktop pages, and 11.00% of mobile. That's up from an average of 1.8% last year, a huge growth factor. It is not hard to see why their popularity is increasing – they offer more design flexibility, and also potentially smaller binary font sizes, especially if multiple styles of the same font are used on the same page.
 
-{# That's huge usage! Any ideas what's driven it? It would appear no variable font has any where near that usage so how come total usage is so high? Has any particular service or platform turned this on by default? #}
+{# TODO - authors That's huge usage! Any ideas what's driven it? It would appear no variable font has any where near that usage so how come total usage is so high? Has any particular service or platform turned this on by default? Discussed at https://github.com/HTTPArchive/almanac.httparchive.org/pull/1816#discussion_r547265540 and @jpamental thinks it's Google Fonts #}
```
Comment (bazzadp) [inline src/content/en/2020/fonts.md:327]: @jpamental you were going to add a sentence or two here.

File: src/content/en/2020/fonts.md [hunk 6]
```diff
@@ -329,9 +324,9 @@ Lots more information about color fonts, including examples, are available at [c
...
 
-{# TODO - authors Should we add a paragraph or bullet list explaing the common font axis and what they mean before launching into their usage? What is difference between wght and wdth? And between slnt and ital? Web Almanac readers may come here from pother chapters and not be familiar with these terms. Plus I am a little familair and don't know these answers! #}
+{# TODO - authors Should we add a paragraph or bullet list explaing the common font axis and what they mean before launching into their usage? What is difference between wght and wdth? And between slnt and ital? Web Almanac readers may come here from pother chapters and not be familiar with these terms. Plus I am a little familair and don't know these answers! Discussed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1816#discussion_r547266330 and suggestion to add a pargraph with a link to https://variablefonts.io/about-variable-fonts/ #}
```
Comment (bazzadp) [inline src/content/en/2020/fonts.md:329]: @jpamental you were going to add a sentence or two here.

**Review by bazzadp — COMMENTED** [review]
Highlighted the remaining TODOs below

**Comment by bazzadp** [comment]
Any chance to look at this yet @jpamental ?

**Comment by jpamental** [comment]
OK - just pushed up some edits. Might need a second look at them but I think they're pretty solid @barrypollard Thanks!

**Comment by bazzadp** [comment]
Good stuff @jpamental - thanks for that. Removing unedited tag and will merge and include in the next release.

**Comment by jpamental** [comment]
Thanks Barry! Appreciate your help and patience :)

Cheers,

Jason

Jason Pamental
401.743.4406

jason@rwt.io
http://rwt.io

**Pull request merged**